### DISCUSSION TITLE
Car tool

### DISCUSSION THREAD

**Discussion by helio58** [gtxaspec/wz_mini_hacks#702] [General]
Hi thanks for this great tool. I m new  using it.
When I use car tool I get a error :" 404 Not Found The requested URL was not found " . wyzecam v3 
Is there  asetting i m missing thanks.

**Comment by virmaior** [comment]
right now it's: `/car/car.html`

I'l fix the url error shortly...

**Reply by helio58** [reply]
Thanks , One question is possible to change between continuos recording and motion recordings ? thanks
